docs/sphinx/quickstart.rst

```diff
@@ -0,0 +1,145 @@
+Quickstart 
+==========
+
+This guide shows you how to install Elasticsearch and perform basic operations 
+with it by using the Python client.
+
+Requirements
+------------
+
+- `Python <https://www.python.org/>`_ 3.6 or newer
+- `pip <https://pip.pypa.io/en/stable/>`_
+
+
+Installation
+------------
+
+To install the latest version of the client, run the following command:
```

Comment:
Not sure I would mention "the latest version of", because if an old version is already installed, it will stay there. The `--upgrade` flag would avoid this, but it seems like an unnecessary complication at this point, hence why I'm suggesting removing "the latest version of".